### DISCUSSION TITLE
RTMP Server Related

### DISCUSSION THREAD

**Discussion by pankajsaini1987** [bluenviron/mediamtx#2411] [Questions]
Hi, i have a h.264 IP Camera, where RTMP option is available 
i want to take that stream from Ip camera to RTSP Server and want to publish that stream to webrtc or rtsp

(like youtube provide us RTMP link and key salt which we copy and paste into the camera and my camera start streaming to youtube)

Now i want to push camera stream by RTMP to your RTSP server. is that possible in your App.

please suggest

**Comment by pankajsaini1987** [comment]
can anyone help?

**Comment by aler9** [comment]
Hello, if your camera works with RTMP links from YouTube, you can set it up by following instructions in the README related to "RTMP clients":

https://github.com/bluenviron/mediamtx#rtmp-clients

Once the stream is ready, conversion to WebRTC or RTSP is automatic.

[a comment by an automated account is not shown]
